CRM/Case/BAO/Case.php

```diff
@@ -1093,7 +1093,8 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
         $caseActivity['no_attachments'] = count($attachmentIDs);
       }
 
-      $caseActivities[$caseActivityId]['links'] = CRM_Case_Selector_Search::addCaseActivityLinks($caseID, $contactID, $userID, $context, $dao);
+      $caseActivities[$caseActivityId]['links']
+        = CRM_Case_Selector_Search::addCaseActivityLinks($caseID, $contactID, $userID, $context, $dao);
```

Comment:
>This is currently failing tests... because?

Seems like jenkin failed to apply the patch. Can you remove this line change as it looks like there is no real change in this part except a next line added for the assignment? If I try to grab a patch of this PR and apply it on my local, i get the same error.

    $ git apply 15135.patch
    error: patch failed: CRM/Case/BAO/Case.php:1093
    error: CRM/Case/BAO/Case.php: patch does not apply

Reply:
@jitendrapurohit I have rebased. The first two commits are #14349, it's only the 3rd commit that should be making changes here.